RadarSDK/RadarUser.h

```diff
@@ -145,6 +146,16 @@ typedef NS_ENUM(NSInteger, RadarLocationSource);
  */
 @property (assign, nonatomic, readonly) BOOL proxy;
 
+/**
+ A boolean indicating whether the user's IP address is being mocked or simulated. May be `false` if Fraud is not enabled.
+ */
+@property (assign, nonatomic, readonly) BOOL mocked;
+
+/**
+ Learned fraud state for the user. May be nil if Fraud is not enabled
```

Comment:
nit: `nil` should have single quotes around it as it does for other properties (ex. `segments`, `topChains`)

Reply:
Done.